Description: rebase project subdir to use project name
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-04-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bulma.sass
+++ b/bulma.sass
@@ -1,10 +1,10 @@
 @charset "utf-8"
 /*! bulma.io v0.9.4 | MIT License | github.com/jgthms/bulma */
-@import "sass/utilities/_all"
-@import "sass/base/_all"
-@import "sass/elements/_all"
-@import "sass/form/_all"
-@import "sass/components/_all"
-@import "sass/grid/_all"
-@import "sass/helpers/_all"
-@import "sass/layout/_all"
+@import "bulma/utilities/_all"
+@import "bulma/base/_all"
+@import "bulma/elements/_all"
+@import "bulma/form/_all"
+@import "bulma/components/_all"
+@import "bulma/grid/_all"
+@import "bulma/helpers/_all"
+@import "bulma/layout/_all"
